siriuspy/siriuspy/sofb/orbit.py

```diff
@@ -673,25 +673,25 @@ def _update_time_vector(self, delay=None, duration=None, channel=None):
         """."""
         if not self.isring:
             return
-        dl = (delay or self.timing.delay or 0.0) * 1e-6  # from us to s
+        dly = (delay or self.timing.delay or 0.0) * 1e-6  # from us to s
         dur = (duration or self.timing.duration or 0.0) * 1e-6  # from us to s
         channel = channel or self.bpms[0].acq_type or 0
 
         # revolution period in s
         if channel == _csbpm.AcqChan.Monit1:
-            dt = self.bpms[0].monit1period
+            dtim = self.bpms[0].monit1period
```

Comment:
you think _dtim_ is better than _dtime_ I used the the standardization PR? if so, I can change all other places and use it. I have no strong opinion about this, other than trying to use the name name.

Reply:
`dtime` is much better! I'll change it!

Reply:
done!